spaceship.zsh

```diff
@@ -309,6 +314,23 @@ spaceship_swift_version() {
   fi
 }
 
+# GOLANG
+# Show current version of golang.
+spaceship_golang_status() {
+  [[ $SPACESHIP_GOLANG_SHOW == false ]] && return
+
+  $(command go version > /dev/null 2>&1) || return
```

Comment:
Better to check whole command and not only subcommand. In case with GVM, better to check `gvm` for existence.